src/Core/Extensions/EmoteExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Discord;
+
+namespace BrackeysBot
+{
+    public static class EmoteExtensions

```

Comment:
Out of curiosity, what are these extensions for?
Is there nothing built-in?

Reply:
The API has 3 kinds of emotes; 
Regular emotes the user can add like :Kappa:
Unicode emojis available by default like :rage: 
Animated emotes the user can add like :ablobrage:
Regular and Animated are handled as Emote : IEmote where unicode emoji are handled as Emoji : IEmote. IEmote only provides GetName() method. As such I had to add this extension to get them easily parseable.